getting_started/first_3d_game/06.jump_and_squash.rst

```diff
@@ -236,13 +236,13 @@ With this code, if no collisions occurred on a given frame, the loop won't run.
    func _physics_process(delta):
         #...
 
-        # Iterate through all collisions that occurred this frame
-        # in C this would be for(int i = 0; i < collisions.Count; i++)
+        # Iterate through all collisions and sliding movements that occurred this frame
```

Comment:
I did some debugging and found out that each time I slide on the ground, the get_slide_collision(index) returned the ground and 2 null object, so I believe this function return the collisions and the movements, and updated the comment accordingly.